packages/easysearch:core/tests/unit/core/index-tests.js

```diff
@@ -15,14 +15,18 @@ Tinytest.addAsync('EasySearch - Unit - Core - Index', function (test, done) {
 
   index.config.engine.search = function (searchDefinition, options) {
     test.equal(searchDefinition, 'testSearch');
-    test.equal(options.search, { limit: 200, skip: 20, props: {} });
+    test.equal(options.search.limit, 200);
+    test.equal(options.search.skip, 20);
+    test.equal(options.search.props, {});
   };
 
   index.search('testSearch', { limit: 200, skip: 20 });
 
   index.config.engine.search = function (searchDefinition, options) {
     test.equal(searchDefinition, 'testSearch');
-    test.equal(options.search, { limit: 10, skip: 0, props: { custom: 'property' } });
```

Comment:
please revert these test fixes as this doesn't ensure anymore that the options have unwanted properties set